pennylane/operation.py

```diff
@@ -262,6 +262,62 @@ def _matrix(*params):
         """
         raise NotImplementedError
 
+    @property
+    def matrix(self):
+        r"""Matrix representation of an instantiated operator
+        in the computational basis.
+
+        **Example:**
+
+        >>> U = qml.RY(0.5, wires=1)
+        >>> U.matrix
+        >>> array([[ 0.96891242+0.j, -0.24740396+0.j],
+                   [ 0.24740396+0.j,  0.96891242+0.j]])
+
+        Returns:
+            array: matrix representation
+        """
+        return self._matrix(*self.parameters)
+
+    @classmethod
+    def _eigvals(cls, *params):
+        """Eigenvalues of the operator in the computational basis.
+
+        This is a *static method* that should be defined for all
```

Comment:
Class method?

Reply:
Just curious what the resolution was here. Above it says `@classmethod` in the code, but here it has static method in the docstring

Reply:
Same question for `matrix`/`_matrix` docstring